code/main.py

```
@@ -0,0 +1,122 @@
+    merged_df = pd.merge(job_data_df, requirements_df, on="jobID", how="inner")
+    return merged_df
+
+# Step 2: Generate Embeddings for Text Fields
+def generate_embeddings(df):
+    print("Generating embeddings...")
+    model = SentenceTransformer('all-MiniLM-L6-v2')
+    text_fields = ["jobTitle", "job function", "industries"]
+    for field in text_fields:
+        print(f"Generating embeddings for {field}...")
+        df[f"{field}_embedding"] = df[field].apply(
+            lambda x: model.encode(str(x)).tolist()
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)